app/controllers/workflow_repository_controller.rb

```diff
@@ -16,10 +16,75 @@ def self.display_methods
     %w[workflows]
   end
 
+  def self.custom_display_modes
+    %w[output]
+  end
```

Comment:
I saw this change in the other PR - Can we discuss, as I don't understand it.

Reply:
This is just used to display the refresh output screen above (last screenshot in the pr description). It changes the @display value to output so the haml file can render a different screen.